build.rs

```diff
@@ -48,7 +48,7 @@ fn main() {
     if cuda {
         println!("cargo:rerun-if-changed=target/perf-libs/libcuda-crypt.a");
         println!("cargo:rustc-link-lib=static=cuda-crypt");
-        println!("cargo:rustc-link-search=native=/usr/local/cuda/lib64");
+        println!("cargo:rustc-link-search=native=/usr/local/cuda-9.2/lib64");
```

Comment:
I guess I don't understand why we would point always at 9.2, why not at the symlink to the version the user has/wants?

Reply:
Seems like fetch-perf-libs.sh maybe should just do a check to see if the expected version is being symlinked to and then error or warn if it's not.

Reply:
Yeah true.  I guess I was assuming that we wouldn't support any random CUDA version but instead require a specific version (9.2 now, 10 soon).  Much like how we lock down other dependencies like rustc, etc

Reply:
> Seems like fetch-perf-libs.sh maybe should just do a check ...

Oh yeah it does that already.  But say we move to CUDA 10 on master but stay with CUDA 9.2 for v0.10 we now have that pesky /usr/local/cuda symlink that's wrong half the time in CI

Reply:
Maybe there's a better way?

Reply:
Well I don't know if support is the right word, but I would like to let people try with the cuda they have before forcing them to upgrade. Cuda is a 2GB+ download which could require a new built kernel modules and video driver which can seriously mess up your system. It's a bit different than just downloading a 10kb user-mode crate and recompiling it with very little risk of side-effects.

Reply:
How about have CI call something like the update-default-cuda.sh script each time, for master it updates the symlink to cuda-10, for release it updates to cuda-9.2 for instance.

Or have an env variable that it checks but defaults to /usr/local/cuda if it isn't set.